### DISCUSSION TITLE
[Esercitazione 15 Novembre 2023] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#194] [Esercizi - Viaggi/Piazza] [answered]
**Esercizio 3**
- **(1)** Determinare $\mathcal{U}(\mathbb{Z}_{15})$
- **(2)** Per ogni $a\in \mathcal{U}(\mathbb{Z}_{15})$ calcolare $a^{1347}$

**Comment by Elia-Belli** [comment, marked as answer]
## Soluzione Ufficiale
![Screenshot 2023-11-16 103629](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/6657515f-3815-4b47-8a93-98e3fb8ea7db)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/4cd52860-88e7-480d-8517-3efe09caae95)
